Server/src/Server.Api/Controllers/LibrariesController.cs

```diff
@@ -55,6 +56,32 @@ public async Task<ActionResult> AddBook(Guid id, AddBookToLibraryCommand newBook
         return Ok();
     }
 
+    [Authorize("Admin")]
```

Comment:
Wydaje mi się, że to może nie być potrzebne. W Adminie ta zakładka 'Oferta' bierze libraryId z claimów, więc dla Admina ten widok się wywali, no chyba że chcemy zostawić ew. dla Admina API dostępne.

Reply:
Tu raczej takie zabezpieczenie 'dla zasady'